temba/settings_common.py

```diff
@@ -1248,3 +1248,9 @@
 
 # Credits expiration config
 CREDITS_EXPIRATION = False
+
```

Comment:
All the changes on this file should be unconsidered since this part would be made on Mailroom/GoFlow side.